pkg/options/unity.go

```diff
@@ -0,0 +1,35 @@
+package options
+
+import "github.com/bugsnag/bugsnag-cli/pkg/utils"
+
+// UnityLineMapping is used to specify options for uploading a Unity IL2CPP mapping file.
+type UnityLineMapping struct {
+	NoUploadIl2cppMappingFile bool       `name:"no-upload-il2cpp-mapping-file" help:"Do not upload the IL2CPP mapping file (LineNumberMappings.json)"`
```

Comment:
Sorry for late change, but I notice that we don't usually say "path" or "file" in equivalent options, so how about `il2cpp-line-mappings` and `no-upload-il2cpp-line-mappings` as the option names?